pages/components/avatar/avatar.html

```diff
@@ -0,0 +1,127 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<meta charset="UTF-8" />
+		<meta http-equiv="X-UA-Compatible" content="IE=edge" />
+		<meta name="viewport" content="width=device-width, initial-scale=1.0" />
```

Comment:
Suggestion: You can add your name in meta tags, this gives a flavor of uniqueness and owing codes.